src/UITests/LiveMode.cs

```diff
@@ -28,13 +27,29 @@ public void LiveModeTests()
         }
 
         private void TestTestResults()
+        {
+            RunTests();
+            ScanAccessibility();
+            ValidateTestModeTitle();
+            driver.TestMode.AutomatedChecks.ValidateAutomatedChecks(12);
+            driver.TestMode.AutomatedChecks.GoToAutomatedChecksElementDetails(4);
+            ValidateResultsInUIATree();
+        }
+
+        private void ValidateResultsInUIATree()
+        {
+            driver.TestMode.ResultsInUIATree.ValidateResults(false, 2, 11);
```

Comment:
>driver [](start = 12, length = 6)

Code smell: all of the lines in this method start "driver.TestMode"--that suggests that perhaps the ValidateResultsInUIATree method belongs in the TestMode class.

Reply:
The parameters passed to the methods called in ValidateResultsInUIATree are specific to this test class and I don’t expect to reuse that exact set of calls often. ValidateResultsInUIATree only exists because I thought having the grouping makes TestTestResults read a little better. 
Does that explanation change your mind, or would you still say this makes more sense in TestMode?

Reply:
I'm fine with it either way.

Reply:
I have merged this as-is because John is out until next week - this potential change can come in separately if we'd like